zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/AuthorizationCreateAuthorizationTest.java

```diff
@@ -128,6 +129,89 @@ public void shouldBeForbiddenToCreateAuthorizationIfNoPermissions() {
             "Insufficient permissions to perform operation 'CREATE' on resource 'AUTHORIZATION'");
   }
 
+  @Test
+  public void shouldBeAuthorizedToCreatePropertyBasedAuthorizationWithDefaultUser() {
```

Comment:
❓ Can you explain the value of this and the test below? Permissions are not so granular that we can give, say, authorization to create authorizations with a property or ID matcher. You can either create or not create authorizations, regardless of the resource matcher. Or did I misunderstand?